fasjson_client/errors.py

```diff
@@ -0,0 +1,17 @@
+class BaseError(Exception):
+  def __init__(self, message, code, data=None):
+    super(BaseError, self).__init__(message)
+    self.message = message
+    self.code = code
+    self.data = data
+
+  def __repr__(self):
+    return f'<{self.__class__.__name__} code={[self.code]} message={self.message} data={self.data}>'
```

Comment:
Do you want the code to appear as a list? If the brackets are only for decoration then I think they should be outside the curly braces.

Reply:
ah not really, it was an error of mine, I will remove it